docs/how-to/hip_runtime_api/memory_management/host_memory.rst

```diff
@@ -0,0 +1,205 @@
+.. meta::
+  :description: This chapter describes the host memory of the HIP ecosystem
+                ROCm software.
+  :keywords: AMD, ROCm, HIP, host memory
+
+.. _host_memory:
+
+********************************************************************************
+Host memory
+********************************************************************************
+
+Host memory where on the host (e.g. CPU) of the machine in random access memory
+(RAM). The host memory has three different main types in HIP:
+
+* Pageable memory
+* Pinned memory
+* Unified Memory
+
+These different types of memory should be used at different use cases. The
+pageable and pinned memory using explicit memory management, where the
+developers have direct control over memory operations, while at the unified
+memory case the developer gets a simplified memory model with less control over
+low level memory operations.
+
+The data transfer differences between the pageable or pinned memory usage
+represented in the next figure.
+
+.. figure:: ../../../data/how-to/hip_runtime_api/memory_management/pageable_pinned.svg
+
+The unified memory management and explicit memory management main difference
+highlighted at the following figure.
+
+.. figure:: ../../../data/how-to/hip_runtime_api/memory_management/unified_memory/um.svg
+
+The unified memory management described at the :doc:`/how-to/hip_runtime_api/memory_management/unified_memory`.
```

Comment:
```suggestion
Unified memory management is described further in :doc:`/how-to/hip_runtime_api/memory_management/unified_memory`.
```
But again, unified memory belongs into its own chapter. The reference would belong to the memory management landing page